code/game/gamemodes/cult/runes.dm

```diff
@@ -360,7 +360,7 @@ var/list/teleport_runes = list()
 	var/turf/T = get_turf(src)
 	for(var/mob/living/M in T)
 		if(!iscultist(M))
-			myriad_targets = M
+			myriad_targets |= M
```

Comment:
Turns out removing that made it so the rune didn't work at all. No idea at all why. Added it back.